packages/core/scripts/setup/patches/patch.ts

```diff
@@ -0,0 +1,177 @@
+import EmbraceLogger from '../../../src/logger';
+import { getMainApplicationPatchable } from '../../util/android';
+import { FileUpdatable } from '../../util/file';
+import {
+  EMBRACE_IMPORT_OBJECTIVEC,
+  EMBRACE_INIT_OBJECTIVEC,
+  getAppDelegateByIOSLanguage,
+} from '../../util/ios';
+
+import {
+  addLineAfterToTextInFile,
+  addLineBeforeToTextInFile,
+  ANDROID_LANGUAGE,
+  IOS_LANGUAGE,
+  MAIN_CLASS_BY_LANGUAGE,
+  SUPPORTED_LANGUAGES,
+} from './common';
+
+export const EMBRACE_IMPORT_SWIFT = 'import Embrace';
+
+export const EMBRACE_INIT_SWIFT =
+  'Embrace.sharedInstance().start(launchOptions: launchOptions, framework:.reactNative)';
```

Comment:
I would put these strings directly in the `textToAdd` definitions below, it's a bit harder to see the patch that's being done when they're defined separately here

Reply:
Do you mean direclty in the object ? 
textToAdd: `Embrace.sharedInstance().start(launchOptions: launchOptions, framework:.reactNative)\n`,

And copy this text and paste into the install/uninstall tests?

Reply:
ya if they are only used in tests I think it's better to put it directly into `textToAdd` here and then just have the values literally written out in the tests